articles/service-fabric/service-fabric-dnsservice.md

```diff
@@ -241,6 +241,21 @@ public class ValuesController : Controller
     }
 }
 ```
+## Recursive Queries
+
+For DNS names that DNS Service cannot resolve on its own (e.g. for a public DNS name), it will forward the query to pre-existing recursive DNS servers on the nodes. 

```

Comment:
```suggestion
For DNS names that the DNS service can't resolve on its own (for example, a public DNS name), it will forward the query to pre-existing recursive DNS servers on the nodes. 

```